src/main/osd/osd.c

```diff
@@ -291,7 +301,7 @@ void pgResetFn_osdConfig(osdConfig_t *osdConfig)
     osdConfig->esc_temp_alarm = ESC_TEMP_ALARM_OFF; // off by default
     osdConfig->esc_rpm_alarm = ESC_RPM_ALARM_OFF; // off by default
     osdConfig->esc_current_alarm = ESC_CURRENT_ALARM_OFF; // off by default
-    osdConfig->core_temp_alarm = 70; // a temperature above 70C should produce a warning, lockups have been reported above 80C
+    osdConfig->core_temp_alarm = CORE_TEMPERATURE_THRESHOLD;
```

Comment:
So, this should be configurable, right? Is it already?

Reply:
yes, already configurable, but not from target.h

This is setting the _default_ value which, as of this change, can be set on a per-target base.

This PR adds a per-MCU series default for the H7 processors that run at higher clock speeds and generally run hotter on FC-sized PCBs (all my prototype boards and the larger FC tester H7 board all run at similar temps).

It's unknown (to me, or anyone else simply reading the existing comment) if the 70/80C applied to F1/F3/F4/F7/etc.